pennylane/workflow/_capture_qnode.py

```diff
@@ -13,6 +13,98 @@
+and reshape the outputs from measurements on the device when multiple measurements are present.
+
+**Gradients other than default qubit backprop**. We managed to get backprop of default qubit for
+free, but no other gradients methods have support yet.
+
+*MCM methods other than single branch statistics*.  Mid circuit measurements
+are only handled via a "single branch statistics" algorithm, which will lead to unexpected
+results. Even on analytic devices, once branch will be randomly chosen on each execution.
+Returning measurments based on mid circuit measurements, ``qml.sample(m0)``,
+is also not yet supported on default qubit or lightning.
+
+>>> @qml.qnode(qml.device('default.qubit', wires=1))
+>>> def circuit(x):
+...     qml.H(0)
+...     m0 = qml.measure(0)
+...     qml.cond(m0, qml.RX, qml.RZ)(x,0)
+...     return qml.expval(qml.Z(0))
+>>> circuit(0.5), circuit(0.5), circuit(0.5)
+(Array(-0.87758256, dtype=float64),
+Array(1., dtype=float64),
+Array(-0.87758256, dtype=float64))
+>>> qml.capture.disable()
+>>> circuit(0.5)
+np.float64(0.06120871905481362)
+>>> qml.capture.enable()
+
+*Device preprocessing and validation*.  No device preprocessing and validation will occur. The captured
+jaxpr is directly sent to the device, whether or not the device can handle it.
+
+>>> @qml.qnode(qml.device('default.qubit', wires=3))
+... def circuit():
+...     qml.Permute(jax.numpy.array((0,1,2)), wires=(2,1,0))
+...     return qml.state()
+>>> circuit()
+MatrixUndefinedError:
+
+*Transforms are still under developement*.  No transforms will currently be applied as part of the workflow.
+
+*Breaking vmap/ parameter broadcasting into a non-broadcasted state*. The current workflow assumes
+that the device execution can natively handled broadcasted parameters. vmap and parameter broadcasting
+will not work with devices other than default qubit.
+
+>>> @qml.qnode(qml.device('lightning.qubit', wires=1))
+... def circuit(x):
+...     qml.RX(x, 0)
+...     return qml.expval(qml.Z(0))
+>>> jax.vmap(circuit)(jax.numpy.array([1.0, 2.0, 3.0]))
+TypeError: RX(): incompatible function arguments. The following argument types are supported:
+    1. (self: pennylane_lightning.lightning_qubit_ops.StateVectorC128, arg0: list[int], arg1: bool, arg2: list[float]) -> None
+    2. (self: pennylane_lightning.lightning_qubit_ops.StateVectorC128, arg0: list[int], arg1: list[bool], arg2: list[int], arg3: bool, arg4: list[float]) -> None
+
+*Grouping commuting measurements and/ or splitting up non-commuting measurements.* Currently, each
+measurment is fully independent and generated from different raw samples than every other measurement.
+To generate multiple measurments from the same samples, we need a way of denoting which measurements
+should be taken together.  A "Combination measurement process" higher order primitive, or something like it.
+We will also need to figure out how to implement splitting up a circuit with non-commuting measuremets into
```

Comment:
```suggestion
should be taken together. A "Combination measurement process" higher order primitive, or something like it.
We will also need to figure out how to implement splitting up a circuit with non-commuting measurements into
```

Reply:
why does measurements have to be such a stupidly difficult word to spell at speed 😬